languages/csharp-dotnet-2.2/template/src/pipe.cs

```diff
@@ -0,0 +1,16 @@
+using Algorithmia;
+using __ALGO__;
+// This file takes your __ALGO__.cs file, loads the apply function and creates a Pipe wrapper around it.
+// WARNING: Do not remove this file from your project, it will make your algorithm unusable.
+
+namespace __ALGO__
+
+{
+    public class pipe
```

Comment:
should this be a capital Pipe?

Reply:
It could be? Honestly the class conventions here get weird... I'm not a big fan of capitalizing everything. But just for consistency I can see a case

Reply:
resolved in latest commit.